lib/rubocop/rspec/config_formatter.rb

```diff
@@ -4,15 +4,15 @@ module RuboCop
   module RSpec
     # Builds a YAML config file from two config hashes
     class ConfigFormatter
-      NAMESPACE = 'RSpec'.freeze
+      NAMESPACES = ['RSpec'.freeze, 'FactoryGirl'.freeze].freeze
 
       def initialize(config, descriptions)
         @config       = config
         @descriptions = descriptions
       end
 
       def dump
-        YAML.dump(unified_config).gsub(/^#{NAMESPACE}/, "\n#{NAMESPACE}")
+        YAML.dump(unified_config).gsub(/^(#{NAMESPACES.join('|')})/, "\n\\1")
```

Comment:
I wonder if `Regexp.union` would have been usable here. Not requesting the change just curious